docs/installation/est/Installing_EST_pkispawn.adoc

```diff
@@ -104,6 +104,59 @@ Additionally, java driver for PostgreSQL need to be installed in the EST server
 # ln -s /usr/share/java/ongres-stringprep/stringprep.jar /usr/share/pki/server/common/lib/
 ```
 
+=== TLS mutual authentication ===
+
+The above configurations of *LDAP* or *PostgreSQL* allow client
+authentication using _username/password_. In some cases or for
+specific operation, such as re-enroll for a new certificate, mutual
+authentication with client certificate is required.
+
+Realm configuration already support certificate based authentication
+out-of-the-box but in order to authenticate a user some additional
+information are needed. In more detail, the user entry has to include
+a _description_ containing some certificate details and the binary
+certificate.
+
+The _description_ has the format
+`<Version>;<Serial>;<Issuer>;<subject>`. The version is the hex value
+(without _0x_), and the serial is in decimal. As an example, if the user
+has a certificate with the following values:
```

Comment:
I think we should also mention that the components in the subject DN and issuer DN are inverted (compared to OpenSSL).